src/file_utils.cr

```diff
@@ -480,7 +447,7 @@ module FileUtils
   #
   # NOTE: Alias of `Dir.rmdir`
   def rmdir(path : String) : Nil
```

Comment:
This method can be deprecated too. I don't know how the core members sees the future of this module, but we can start now removing unnecessary aliases at least.

Reply:
I think is valid to have this unix-isms in a module. And FileUtils offers some operations that I think does not fit neatly in Dir or File.

If you say this can be deprecated because is a plain alias, then I don't agree. My intention is to keep FileUtils with similar methods and semantics as you would find in a unix shell.

I think @maiha has some opinions regarding why unix-isms are nice.

Reply:
I was only pointing duplicated methods. There are lots of terminal methods missing, which are very useful and very commonly used like `ls`, `head`, `tail`, `wc`, `find`, etc...

I don't think they will be accepted (will they?), because all aliases at the end can be better written in more idiomatic Crystal.

Reply:
Anyway, sorry it starts to be out of scope :sweat_smile: .
Let's keep it then.

Reply:
This is not a "plain alias" as @bcardiff says, nor is it a "duplicate method".

Since Crystal is an "object-oriented" language, the core classes such as `File` and `Dir`, their APIs are provided in an object-oriented paradigm.

On the other hand, since UNIX is a "procedural" concept, it has different APIs.

Here, the two paradigm API entry points are each defined and their **implementations** are only shared.

So, if you want to clean up, I think it's a good idea to remove the confusing annotation part of "NOTE: Alias of".

Reply:
Both are called in a functional way – they are class methods, not instance methods. Only the name change.

I think `FileUtils` is just there because of Ruby, a reason why I believe this is nobody asked to add `ls`, which is IMO of the most common and useful command.

`Utils` is also quite bad, if the main purpose of the module is to provide a command-like interface, it would better be called `FileCommands`or `Shell`.

Reply:
Both are called in a functional way – they are class methods, not instance methods. Only the name change.

I think `FileUtils` is just there because of Ruby, a reason why I believe this is nobody asked to add `ls`, which is one of the most common and useful command.

`Utils` is also quite bad, if the main purpose of the module is to provide a command-like interface, it would better be called `FileCommands`or `Shell`.

Reply:
I don't say everything on FileUtils has to go to trash. I'm only saying we don't have to copy blindly 1-1 the Ruby API, and review the actual module to see what is often used (`cp`, `dir` methods) and those not or confusing like `cmp` or `pwd` (I searched both on GitHub before saying this)

Reply:
>  nobody asked to add ls

Well, why not try to think of `FileUtils` as `File (mutable) Utils`?

Reply:
Members of the core team like FileUtils so it's going to stay. I know opinions are diverse here but the decision is already taken, so please let's not discuss removing anything.